src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public int getRandomBall() {
+        Random randomBall = new Random();
+        int value = randomBall.nextInt(101);
+        return value;
+    }
+
+}
```

Comment:
1. Lottery should provide random ball, not an int
2. Magic number should be constant with proper naming
3. This service should use both Random and ColorSupplier services to make work done
```suggestion
    public Ball getRandomBall() {
        return new Ball(random.nextInt(NUMBER_LIMIT), colorSupplier.getRandomColor().name());
    }
}
```